streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImplTest.java

```diff
@@ -736,42 +705,34 @@ public void shouldProvideCorrectStrings() {
         assertThat(ROLLUP_VALUE, is("all"));
     }
 
-    private void setupRemoveSensorsTest(final Metrics metrics,
-                                        final String level) {
-        final String fullSensorNamePrefix = INTERNAL_PREFIX + SENSOR_PREFIX_DELIMITER + level + SENSOR_NAME_DELIMITER;
-        resetToDefault(metrics);
-        metrics.removeSensor(fullSensorNamePrefix + SENSOR_NAME_1);
-        metrics.removeSensor(fullSensorNamePrefix + SENSOR_NAME_2);
```

Comment:
`setupRemoveSensorsTest` was called only inside the `shouldRemoveThreadLevelSensors`. 

Since these two lines are used only to verify things, they were moved to the `shouldRemoveThreadLevelSensors` where verification should indeed happen:
```java`
        metrics.removeSensor(fullSensorNamePrefix + SENSOR_NAME_1);
        metrics.removeSensor(fullSensorNamePrefix + SENSOR_NAME_2);
```